### PR TITLE
Add myself to the suggested "CODEOWNERS" and "README.md"

### PR DESCRIPTION
**Pull request by j4ck-code**

### CODE REVIEW

**Comment by j4ck-code** [comment]
Lemme know if I made any mistake 😅

File: CODEOWNERS [hunk 1]
```diff
@@ -9,3 +9,4 @@
 # Korean
 ko/ @revi
 pt-br/ @rffontenelle
+bg/ @j4ck-code
```
Comment (mtatko) [inline CODEOWNERS:12]: The file itself looks okay, however I would add some formatting to it, so it just looks better

```suggestion
# Korean
ko/ @revi

# Brasilian Portugese
pt-br/ @rffontenelle

# Bulgarian
bg/ @j4ck-code
```

File: bg/index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    </p>
+    <p>
+      Други сходни проблеми: <a href="https://xyproblem.info/">The XY Problem</a>, <a href="https://nohello.net/">No Hello</a>.
+      Допълнителна информация: <a href="https://stackoverflow.com/help/how-to-ask">How do I ask a good question?</a>,
+      или ако имате повече време: <a href="http://catb.org/~esr/faqs/smart-questions.html">How To Ask Questions The Smart Way</a>.
+    </p>
+  </main>
+  <footer>
+    Съдържанието е главно откраднато от <a href="https://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
+    -
```
Comment (mtatko) [inline bg/index.html:82]: As a translator, you could add a link to your Github profile here. I would recommend doing that to stay consistent with other versions of the website

Comment (j4ck-code) [reply bg/index.html:82]: Sure, just a sec

**Review by mtatko — APPROVED** [review]

**Review by tulir — APPROVED** [review]

**Pull request merged**